adding more staking FAQs (#249)
onflow/flow: docs/content/staking/faq.mdx

We have the same restriction in place for an account address and an active delegation.
This means that 1 account address can have at maximum 1 active stake and 1 active delegation at any given time.
If a token holder wants to have additional stake and/or delegate transactions, then they must create additional account addresses to do so.

> ### Is there a utility to generate the generate the FLOW addresses?

Reviewer: ```suggestion
### Is there a utility to generate the FLOW addresses?
```

Staking or delegation of FLOW has to be done via Flow port which requires a login through Ledger or Blocto.
Hence, a utility cannot be used to generate a Flow address since it cannot be used to stake or delegate via Flow port.

### Is the wallet key transferred to the staked or delegated node?

No- The keys on the node are different from the wallet keys. The wallet keys always stay in the wallet.
A node operator generates the staking and networking keys separately which will be used on the node.

### Can I stake through multiple accounts to meet the minimum FLOW required for staking a node?

No, the minimum staking FLOW should come from a single account for all node type except for the consensus node.
The minimum for consensus nodes can cumulatively come from a combination a staking account and a delegation account.

### How can I reach the Consensus node minimum stake of 500K FLOW

> The consensus node min 500K FLOW can be reached cumulatively with any combination of a lease and delegation. e.g. lease 250K and then delegate the remaining 250K.

Reviewer: ```suggestion
The consensus node min 500K FLOW can be reached cumulatively with any combination of a lease and delegation as long as the lease is a minimum of 250k FLOW. e.g. lease 250K and then delegate the remaining 250K.
```

### Is rewards payout another spork?

No, rewards payout is not a spork but (currently) a separate transaction that we initiate.

### Are leased tokens technically owned by the recipient or are they still considered leased after being staked?

DapperLabs has possession of the private keys for the account where the leased tokens are held, so technically DapperLabs has ownership of them.

> The recipient of the leased tokens however has the ability to stake them for their nodes, but don't have the ability to withdraw them or do anything else with them.

Reviewer: ```suggestion
The recipient of the leased tokens however has the ability to stake them for their nodes and withdraw the rewards from staking, but doesn't have the ability to withdraw them or do anything else with them.
```

> ### How long will tokens be leased to us?

Reviewer: ```suggestion
### How long are tokens leased?
```
